cedar-policy-generators/src/expr.rs

```diff
@@ -268,11 +268,7 @@ impl<'a> ExprGenerator<'a> {
         if self.should_generate_unknown(max_depth, u)? {
             let v = self.generate_value_for_type(target_type, max_depth, u)?;
             let name = self.unknown_pool.alloc(target_type.clone(), v);
-            let unknown_type: Option<ast::Type> = target_type.clone().try_into().ok();
-            match unknown_type {
-                Some(ty) => Ok(ast::Expr::unknown(ast::Unknown::new_with_type(name, ty))),
-                None => Ok(ast::Expr::unknown(ast::Unknown::new_untyped(name))),
-            }
+            Ok(ast::Expr::unknown(ast::Unknown::new_untyped(name)))
```

Comment:
What's the reason for this change?

Reply:
Bumping this question from @khieta as well

Reply:
Reverted, good catch @khieta